test/neo/client/Client.d

```diff
@@ -31,7 +31,7 @@ public class Client
 
     ***************************************************************************/
 
-    private class Neo
+    public class Neo
```

Comment:
Huh. This is not intended to be public. Why is this change required?

Reply:
It is used from `test/neo/main.d`

Reply:
also `test/client_auth/main.d`

Reply:
Ah yes, of course. It's used for the various aliases it contains. I think this should be a case for a private ctor.

Reply:
`private class Blocking` will need the same treatment.

Reply:
Actually, no, sorry, it only contains methods, which can be accessed via the instance.

Reply:
Actually, no, sorry 2: `Blocking` mixes is `TaskBlockingCore`, which contains types which will need to be accessible from outside.

Reply:
Whatever. Can you do any change you find necessary as a separate PR? I am only fixing compiler errors for existing code :P

Reply:
Sure. Can you at least change the ctor of `Neo` to be private in the same commit where you make the class public?

Reply:
It is already private because it is mixed in by `ClientCore`

Reply:
Ah excellent. Thanks for checking.